eng/scripts/Update-Snippets.ps1

```diff
@@ -19,8 +18,20 @@ if ($ServiceDirectory -and ($ServiceDirectory -ne "*")) {
 
 if (-not (Test-Path env:TF_BUILD)) { $StrictMode = $true }
 
-if($StrictMode) {
-    Resolve-Path "$root" | %{ dotnet run -p $generatorProject -b "$_" -sm -c Release }
-} else {
-    Resolve-Path "$root" | %{ dotnet run -p $generatorProject -b "$_" -c Release }
+dotnet tool install Azure.Sdk.Tools.SnippetGenerator `
+-g --add-source "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-tools/nuget/v3/index.json" `
```

Comment:
I assume the plan is to use the azure-sdk-for-net feed once you make your other changes.